src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
```

Comment:
move to `*` selector and delete here and in 
https://github.com/mate-academy/layout_product-cards/pull/3598/files#diff-ace49a3c54516d750511c9dacfcba2b0529f06110018538bac0c2e95510f6e9aR69